### PR TITLE
NETOBSERV-1709 update noo comparison sheets with context

### PR DESCRIPTION
**Pull request by memodi**
- Add NOO Bundle info to replace UUID for comparison perf sheet

Examples:
-  [sheet](https://docs.google.com/spreadsheets/d/1CQvUg3CJbPdhSCX__u_-Kw62F5zld3vrKH75BhsazaQ/edit?gid=949491585#gid=949491585) When using Downstream QE index catalog image. 
- When pre-merge Operator bundle image is used, that value will be used in sheets:
image

### CODE REVIEW

**Comment by memodi** [comment]
/hold 
(testing more)

cc @Amoghrd

File: scripts/queries/netobserv_prometheus_queries.yaml [hunk 1]
```diff
@@ -169,3 +169,15 @@
 # total working set memory by all Kafka pods - per pod is not included here as that is collected above
 - query: sum(container_memory_working_set_bytes{namespace="netobserv", container="", pod=~"kafka.*"})
   metricName: workingsetKafkaTotals
+
+###########################
+# Total NetObserv metrics #
+###########################
+
+# Total NetObserv CPU Usage, assuming Loki and Kafka also runs in netobserv NS
+- query: (sum(pod:container_cpu_usage:sum{namespace=~"netobserv|openshift-netobserv-operator"}) or on() vector(0))
```
Comment (jotak) [inline scripts/queries/netobserv_prometheus_queries.yaml:178]: it doesn't seem to capture netobserv-privileged?
=>
```suggestion
- query: (sum(pod:container_cpu_usage:sum{namespace=~"netobserv.*|openshift-netobserv-operator"}) or on() vector(0))
```

Comment (memodi) [reply scripts/queries/netobserv_prometheus_queries.yaml:178]: thanks for catching this.

**Review by jotak — APPROVED** [review]
lgtm!

**Comment by memodi** [comment]
here's the latest test results where both the uuids are replaced: https://docs.google.com/spreadsheets/d/1F3jT5SlLvYaJxxAxYcZTM1FopGn0p8pHN22y4gKe4I0/edit?gid=1897222801#gid=1897222801

/unhold

**Comment by memodi** [comment]
/hold 
(i'll add login to registry.stage.redhat.io as well)

**Comment by memodi** [comment]
/unhold
@Amoghrd can you take one more pass?

**Pull request merged**